pantry/common/api_common.py

```diff
@@ -7,3 +8,27 @@ def get_columns(parameters, columns):
             cols.append(columns[col])
 
     return cols if len(cols) > 0 else columns
+
+
+def filter_columns(parameters, query, supported_columns):
```

Comment:
Agreed, this is the method that handles url filters like?health_percent={"gt":75}. I will add comments.